cmake/build_configurations/mysql_release.cmake

```diff
@@ -149,7 +149,7 @@ IF(UNIX)
           RedHat/Fedora/Oracle Linux: yum install libaio-devel
           SuSE:                       zypper install libaio-devel
 
-          If you really do not want it, pass -DIGNORE_AIO_CHECK=ON to cmake.
+          If you really do not want it, pass -DIGNORE_AIO_CHECK=YES to cmake.
```

Comment:
Why this change? In `debian/rules` you are adding the value `-DIGNORE_AIO_CHECK=ON`. I believe that in `cmake`, `ON` and `YES` are interchangeable for `BOOL` parameters.

Reply:
It felt unintuitive to turn off some feature using "on".

Reply:
Yes, I find this completely backwards too. I never liked the ‘add a dependency if the header files are present’ approach. The `libaio` dependency should have been ‘opt-in’ in the first place. OK, I agree with your reasoning.